ethicml/data/csvs/make_synthetic_4.py

```diff
@@ -0,0 +1,105 @@
+"""Generate Synthetic data."""
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+
+
+def sigmoid(x: np.ndarray) -> np.ndarray:
```

Comment:
I think you have this function in 4 places now.

Reply:
Is there a reason you've eschewed implementing it in utils as you have the Heaviside function, the sole "Activation"?